wgpu-types/src/lib.rs

```diff
@@ -1200,13 +1610,19 @@ impl<L> RenderBundleDescriptor<L> {
     }
 }
 
+/// How a shader will receive a texture.
```

Comment:
"Type of the data in a texture.

Used to tell shaders what type of data to read from a texture"